test/unit/stream_alert_rule_processor/test_sink.py

```diff
@@ -78,16 +64,17 @@ def test_json_from_dict():
                     'entity': "payload_entity_01"
                 }
             }
-        }}
+        }
 
-        json_message = sink.json_dump(sns_dict)
+        json_message = sink._json_dump(alert)
 
         # Test with single alert entry
-        assert_equal(json_message, '{"default": {"rule_name": "test_rule_01", ' \
-            '"metadata": {"outputs": "rule.outputs_01", "type": "payload_type_01", ' \
-            '"log": "payload_data_01", "source": {"service": "payload_service_01", ' \
-            '"entity": "payload_entity_01"}}, "record": {"record_data_key02_01": ' \
-            '"record_data_value02_01", "record_data_key01_01": "record_data_value01_01"}}}')
+        assert_equal(json_message, '{"record": {"record_data_key02_01": ' \
+            '"record_data_value02_01", "record_data_key01_01": "record_data_value01_01"}, ' \
+            '"metadata": {"source": {"service": "payload_service_01", ' \
+            '"entity": "payload_entity_01"}, "rule_name": "test_rule_01", ' \
+            '"type": "payload_type_01", "log": "payload_data_01", ' \
+            '"outputs": "rule.outputs_01"}}')
```

Comment:
You don't need the line continuation markers (`\`) (this is discouraged by Google Python style guide); Python has implicit string joining.